go.mod

```diff
@@ -2,19 +2,19 @@ module github.com/crc-org/machine-driver-libvirt
 
 require (
 	github.com/crc-org/machine v0.0.0-20221028075518-f9b43442196b
-	github.com/libvirt/libvirt-go v4.5.0+incompatible
-	github.com/sirupsen/logrus v1.9.0
-	github.com/stretchr/testify v1.8.0
-	libvirt.org/go/libvirtxml v1.8004.0
+	github.com/libvirt/libvirt-go v6.0.0+incompatible
```

Comment:
The commit log says " gomod: update to go 1.19 ", but this is doing more than that, imo the main highlight of this change is the update of libvirt-go to 6.0.0